util/kvencoder/kv_encoder.go

```diff
@@ -226,6 +226,7 @@ func (e *kvEncoder) initial(dbName string, idAlloc autoid.Allocator) (err error)
 
 	se.GetSessionVars().IDAllocator = idAlloc
 	se.GetSessionVars().ImportingData = true
+	se.GetSessionVars().SkipUTF8Check = true
```

Comment:
The same as ImportingData. Shouldn't it be set by the user session?

Reply:
This session is only used by the kvEncoder.